ios/FluentUI/Vnext/Drawer/Drawer.swift

```diff
@@ -80,14 +79,14 @@ public struct MSFDrawerView<Content: View>: View {
 
     public var body: some View {
         GeometryReader { proxy in
-            MSFSlideOverPanel(
+            MSFSlideOutPanel(
```

Comment:
What's the reason behind the name change of "SlideOverPanel" to "SlideOutPanel"?

Reply:
To avoid naming collisions with `slideover` presentation style.